### PR TITLE
Address Shee Hui's comments

### PR DESCRIPTION
**Pull request by Bluntsord**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -507,7 +508,7 @@ State: Home Window
 
 **MSS**
 1. User chooses to find a client within DonnaFin using the right syntax.
-2. DonnaFin displays the clients that match the keyword inputted.
+2. DonnaFin displays the clients that match the keyword being input.
```
Comment (sheehui) [inline docs/DeveloperGuide.md:511]: DonnaFin displays the clients that match the input keywords.

**Comment by sheehui** [comment]
the rest looks good!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ This marks the end of the quick start. Click
   e.g. A valid input for `v/$ASSET_VALUE` is `v/$1000`. `v/1000` will not be accepted by DonnaFin.
 </div>
 L
```
Comment (sheehui) [inline docs/UserGuide.md:121]: remove the L

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bharathcs) [reply docs/DeveloperGuide.md:511]:
```suggestion
2. DonnaFin displays the clients that match the input keyword.
```

File: docs/UserGuide.md (hunk 2 above)
Comment (bharathcs) [reply docs/UserGuide.md:121]:
```suggestion

```

**Review by bharathcs — APPROVED** [review]
LGTM!

**Pull request merged**